README.md

```
@@ -50,6 +50,9 @@ make test-basic
 make run
 ```
 
+📖 **[See detailed documentation and full feature guide →](mad_spark_multiagent/README.md)**
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)